src/libsystemd/sd-bus/bus-common-errors.h

```diff
@@ -59,6 +59,8 @@
 #define BUS_ERROR_SLEEP_VERB_NOT_SUPPORTED     "org.freedesktop.login1.SleepVerbNotSupported"
 #define BUS_ERROR_SESSION_BUSY                 "org.freedesktop.login1.SessionBusy"
 #define BUS_ERROR_NOT_YOUR_DEVICE              "org.freedesktop.login1.NotYourDevice"
+#define BUS_ERROR_NOT_CONFIGURED               "org.freedesktop.login1.NotConfigured"
```

Comment:
I'd prefer if this would be a tiny bit more descriptive. Maybe "DesignatedMaintenanceTimeNotConfigured".